src/actions/crud.js

```diff
@@ -80,6 +80,12 @@ export function updateResource({ resource, id, opts }) {
   }));
 }
 
+export function updateCollection({ resource, id, opts }) {
+  return (dispatch) => dispatch(executeFetch({
+    resource, id, opts, actionType: 'UPDATE_COLLECTION',
```

Comment:
This would receive an optional `restAction: 'POST'` argument perhaps.

Reply:
Should this `restAction` be provided in the `opts` parameter? We can extract it at `executeFetch.js` level perhaps here: https://github.com/americanexpress/iguazu-rest/blob/master/src/actions/executeFetch.js#L60

Reply:
Yes `opts` seem the right place for this 👍

Reply:
What do you think about allowing the `method` key to be passed into `opts`? That way, we do not need to ask for additional keys and potentially complicate which key to use.

Reply:
Perhaps something like this if you look at the added test case in this commit: 7f24c7cf44fa295aae6702ab7e131f17f15664ad

Reply:
Yes this works 👍